src/colorizer/Dataset.ts

```diff
@@ -159,9 +160,10 @@ export default class Dataset {
         data: source.getBuffer(),
         min: source.getMin(),
         max: source.getMax(),
-        unit: metadata?.unit || "",
```

Comment:
The `?` handles the case where `metadata=undefined`, which won't happen since `metadata` is a defined input to `loadFeature` here. And yeah, the fallback handles the case where `metadata.unit=undefined`.